### PR TITLE
Bug fix for jazz-ui

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jazz-ui/src/app/pages/service-overview/service-overview.component.ts [hunk 1]
```diff
@@ -620,7 +620,7 @@ export class ServiceOverviewComponent implements OnInit {
         var k=2;
         this.sortEnvArr();
 
-        if(this.environ_arr!=undefined)
+        if(this.environ_arr!=undefined){
```
Comment (svidhani) [inline jazz-ui/src/app/pages/service-overview/service-overview.component.ts]: indentation here and other places as well

Comment (supritat) [reply jazz-ui/src/app/pages/service-overview/service-overview.component.ts]: Done

File: jazz-ui/package-lock.json [hunk 2]
```diff
@@ -1093,7 +1093,7 @@
       "integrity": "sha1-OciRjO/1eZ+D+UkqhI9iWt0Mdm8=",
       "dev": true,
       "requires": {
-        "hoek": "2.16.3"
+        "hoek": "4.2.1"
```
Comment (suryajak) [inline jazz-ui/package-lock.json]: The root parent that uses hoek should be upgraded in package.json and I don't see that change. Do you know the root parent in this case?

File: jazz-ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.ts [hunk 3]
```diff
@@ -10,488 +10,455 @@ import { Subscription } from 'rxjs/Subscription';
...
+        // this.environ_arr[i].status=this.environ_arr[i].status.split(" ").join("\ n")
+        if (this.environ_arr[i].logical_id == 'prd' || this.environ_arr[i].logical_id == 'prod') {
+          this.prodEnv = this.environ_arr[i];
+          continue;
+        }
+        if (this.environ_arr[i].logical_id == 'stg') {
+          this.stgEnv = this.environ_arr[i];
+          continue;
+        } else {
+          if (this.environ_arr[i].status !== 'archived') {
```
Comment (hks93) [inline jazz-ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.ts]: making a note

File: jazz-ui/package.json [hunk 4]
```diff
@@ -48,7 +48,7 @@
     "ts-helpers": "^1.1.1"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.2",
+    "@angular/cli": "^1.7.3",
```
Comment (hks93) [inline jazz-ui/package.json]: please Get these changes reviewed by surya , and chec

File: jazz-ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.ts (hunk 3 above)
Comment (hks93) [inline jazz-ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.ts]: Reviewed the UI related changes works fine: approved 
Please get the angular-cli version change and  protractor version change reviewed by surya

Comment (supritat) [reply jazz-ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.ts]: Done

File: jazz-ui/package-lock.json (hunk 2 above)
Comment (supritat) [reply jazz-ui/package-lock.json]: Worked on it

**Pull request merged**